numpy/ma/tests/test_core.py

```diff
@@ -1990,6 +2001,205 @@ def test_datafriendly_mul_arrays(self):
         assert_equal(a, [[1, 1], [3, 3]])
         assert_equal(a.mask, [[0, 1], [0, 1]])
 
+    def test_inplace_addition_scalar_type(self):
+        # Test of inplace additions
+        for t in self.othertypes:
```

Comment:
Well, you would need to separate out the valid from the invalid. Note that testing for appropriate warnings and errors is also part of testing. That would make for complete tests, although that may be pushing the scope of this PR...